mailbox/api/src/main/java/org/apache/james/mailbox/FlagsBuilder.java

```diff
@@ -23,6 +23,10 @@
 
 public class FlagsBuilder {
 
+    public static FlagsBuilder builder() {
+        return new FlagsBuilder();
+    }
+
     private final Flags internalFlags;
 
     public FlagsBuilder() {
```

Comment:
Not if I don't want to have a big code diff that will be hard to be merged. I would have normaly, but, let's put it this way, the team "have some pressure". Should I change it everywhere ?